### PR TITLE
Map documentation update

### PR DESCRIPTION
**Pull request by zaouchelo**
## Description

The PR aims to fix a wrong url in Graph views > Troubleshooter page.
In addition it reformulate the prerequisite about the centreon map user that needs to be granted access to all ressources.

## Target serie

- [x] 20.04.x
- [x] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

### CODE REVIEW

File: en/graph-views/install.md [hunk 1]
```diff
@@ -194,8 +194,9 @@ configurations are possible.
 
 ### Centreon Web interface
 
-The Centreon MAP server requires a dedicated user who has access to all
-resources. Since the password will be stored in human-readable form in a
+You must provide to Centreon MAP server a dedicated user
+**who has access to all resources** through appropriate access list groups.
```
Comment (cg-tw) [inline en/graph-views/install.md:198]:
```suggestion
**who has access to all resources** through the appropriate [access list groups](../administration/access-control-lists.html). 
```

**Review by cg-tw — CHANGES_REQUESTED** [review]
Can you make the changes in French too?

File: en/graph-views/install.md (hunk 1 above)
Comment (zaouchelo) [reply en/graph-views/install.md]: OK, no problem

**Comment by zaouchelo** [comment]
> Can you make the changes in French too?

OK, no problem

**Review by cg-tw — APPROVED** [review]

**Review by Nohzoh — APPROVED** [review]

**Pull request merged**